config/config.go

```diff
@@ -108,13 +115,6 @@ func (cfg *Config) validateServicePipelines() error {
 			if cfg.Receivers[ref] == nil {
 				return fmt.Errorf("pipeline %q references receiver %q which does not exist", pipeline.Name, ref)
 			}
-
-			// Validate the receiver configuration by the custom configuration validator
-			recCfg := cfg.Receivers[ref]
-			if err := recCfg.Validate(); err != nil {
-				return fmt.Errorf("pipeline %q references receiver %q which has invalid configuration with error: %v", pipeline.Name, ref, err)
-			}
```

Comment:
Do you know why this was not in `(cfg *Config) Validate()` earlier? Is it possible this is somehow important?

Reply:
No specific reason, it was added in the previous commit.